paddle/phi/api/profiler/trace_event.h

```diff
@@ -20,52 +20,64 @@ limitations under the License. */
 
 namespace phi {
 
+#define TRACER_EVENT_TYPES                                                  \
```

Comment:
```suggestion
#define TRACER_EVENT_TYPES(X)                                                 \
```
一般我们推荐把宏X也传入进来。否则必须要求外部定义的处理名字必须是X。在框架中更多的是用下划线 _ 来表示内层宏明